assets/scss/templates/_signup.scss

```diff
@@ -6,9 +6,9 @@
   align-items: stretch;
 
   .main-panel {
-    width: 500px;
+    width: 525px;
     height: 100vh;
-    padding: 40px 50px;
+    padding: 40px $dp-spaces--lv6;
```

Comment:
40px is not standard? maybe we should normalize it.

Reply:
I'm going to normalize it in another PR

Reply:
👍